syncserver/src/server/mod.rs

```diff
@@ -266,6 +273,14 @@ impl Server {
             &Metrics::from(&metrics),
             blocking_threadpool.clone(),
         )?;
+        let glean_logger = Arc::new(GleanEventsLogger {
+            // app_id corresponds to probe-scraper entry.
+            // https://github.com/mozilla/probe-scraper/blob/main/repositories.yaml
+            app_id: "syncstorage".to_owned(),
+            app_display_version: env!("CARGO_PKG_VERSION").to_owned(),
+            app_channel: "prod".to_owned(),
```

Comment:
(or whatever the new env var is named?)
```suggestion
            app_channel:  env!("SYNC_ENVIRONMENT").to_owned(),
```